core/domain/role_services.py

```diff
@@ -209,9 +208,10 @@ def get_role_graph_data():
     role_graph['links'] = []
     role_graph['nodes'] = {}
     for role in PARENT_ROLES:
-        role_graph['nodes'][role] = HUMAN_READABLE_ROLES[role]
-        for parent in PARENT_ROLES[role]:
-            role_graph['links'].append({'source': parent, 'target': role})
+        if role != feconf.ROLE_ID_SUPER_ADMIN:
```

Comment:
This thought had crossed my mind but I would say I am not entirely sure at the moment.

Since it has to inherit all the actions from admin and has some additional actions, assigning a role of admin to gae super user doesn't sound right.

Also, I'll be dealing with actions for these (super admin and admin) in following weeks so I could put more thought on it then.

Since the role is anyway not visible right now and does not has any impact. I think we can deal with it there.

What do you say?